README.md

```diff
@@ -48,20 +49,20 @@ Email configuration, used to access IMAP server
 | ImapServer        | :heavy_check_mark: | Address of imap server with port number ex: _imap.example.com:143_ |
 | Username          | :heavy_check_mark: | Email address or username used authenticate on email server        |
 | Password          | :heavy_check_mark: | Password used authenticate on email server                         |
-| StartTLS          | false              | Enable StartTLS connection if server supports                      |
-| TLSAcceptAllCerts | false              | Accept insecure certificates with TLS connection                   |
+| StartTLS          | `false`            | Enable StartTLS connection if server supports                      |
+| TLSAcceptAllCerts | `false`            | Accept insecure certificates with TLS connection                   |
 
 #### Mattermost
 
 Mattermost configuration
 
 | Field    | Required/Default   | Description                                                                                                                |
 |----------|:------------------:|----------------------------------------------------------------------------------------------------------------------------|
-| Server   | :heavy_check_mark: | Address of mattermost server. Please inform protocol and port if its necessary ex: _<https://mattermost.example.com:8065>_ |
+| Server   | :heavy_check_mark: | Address of Mattermost server. Please inform protocol and port if its necessary ex: _<https://mattermost.example.com:8065>_ |
 | Team     | :heavy_check_mark: | Team name. You can find teams name by [(URL)](https://github.com/stuvusIT/mattermost-mattermail#teamchannel)               |
```

Comment:
Of course, the link doesn't work yet. But it also will not work after merging, because this is the only occurence of `teamchannel` in the raw readme file.